src/pfe/file-watcher/server/src/utils/kubeutil.ts

```diff
@@ -227,23 +227,26 @@ export async function getFilesOrFoldersInContainerWithTimestamp(projectID: strin
                 logger.logInfo("No files were found");
                 return [];
             } else {
-            logger.logInfo("At least one file was found");
-            const files = data.stdout.replace(/\n/g, " ").replace(/[^ -~]+/g, "").split(" ").filter((entry: string) => { return entry.trim() != ""; });
-            const fileIndex = 8; // file is the 9th argument from the above command
-            const filesTimestamp: Array<logHelper.LogFiles> = [];
-            for (let index = fileIndex; index < files.length; index = index + fileIndex + 1) {
-                const file = files[index];
-                const dateString = files[index - 3] + " " + files[index - 2] + " " + files[index - 1];
-                if (!folderName) {
-                filesTimestamp.push({file: path.join(fileLocation, file), time: +new Date(dateString)});
-                } else {
-                if (file.toLowerCase() === folderName.toLowerCase()) {
-                    filesTimestamp.push({file: path.join(fileLocation, folderName), time: +new Date(dateString)});
-                    break;
-                }
+                logger.logInfo("At least one file was found");
+                const lines = data.stdout.split("\n");
+                const fileIndex = 8; // file is the 9th argument from the above command
+                const filesTimestamp: Array<logHelper.LogFiles> = [];
+                for (let line of lines) {
+                    line = line.replace(/[^ -~]+/g, "").replace(/\s+/g, " ").trim();
+                    if (line != "") {
```

Comment:
Can you explain what changes are being made here?

Reply:
Same as dockerutils.